packages/fluent-ui/test/__snapshots__/Array.test.tsx.snap

```diff
@@ -167,38 +141,12 @@ exports[`array fields checkboxes 1`] = `
     </div>
   </div>
   <div>
-    <br />
-    <div
-      className="ms-Grid-col ms-sm12"
+    <button
+      className="btn btn-info"
+      type="submit"
```

Comment:
I think this is a problem, we want to use the regular fluent-ui button, but it seems to have been replaced by bootstrap now.

Reply:
same with all the other themes

Reply:
btn btn-info that is the default className

Reply:
should I remove the default className